internal/pkg/metrics/manifest.go

```diff
@@ -1,10 +1,19 @@
 package metrics
 
-import "github.com/kyma-project/lifecycle-manager/pkg/queue"
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	ctrlmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	"github.com/kyma-project/lifecycle-manager/pkg/queue"
+)
 
 type ManifestRequeueReason string
 
 const (
+	metricReconcileDuration               string                = "reconcile_duration_seconds"
```

Comment:
Can you add a test case to the E2E kyma_metrics_test that this metric has a non-zero value?

Reply:
oh. thanks for the info, didn"t know of this test. will implement